### ISSUE TITLE
Not Delivered. Tap to try again 

### ISSUE THREAD

**Issue by osamasayed585** [hexascribe/chatbot-builder#3]
**Description**
I am facing an issue where the messages in the chatbot library are not being delivered successfully. After sending a message, I immediately receive a notification saying "Not Delivered. Tap to try again."

**Steps to Reproduce**
- [ ] Obtain the OpenAI API Key from the OpenAI Platform Account.
- [ ] Add the `mavenCentral()` repository.
- [ ] Include the dependency for the chatbot library.
- [ ] Implement the following Kotlin code snippet:

```Kotlin
@Composable
fun ChatScreen() {
    val chatbot = ChatBot.Builder("MY_OPEN_AI_KEY")
        .setDarkMode(true)
        .addMessage(RoleEnum.ASSISTANT, "Hi, how can I help you today?")
        .addPreSeededMessage(RoleEnum.SYSTEM, "You are a helpful seller car assistant")
        .setInputFieldBorderWidth(1)
        .build()
    chatBot.ComposeScreen()
}
```

Refer to the Android sample provided.
Add the necessary internet permission in the app's manifest file.
Run the application on the Android Studio Emulator with Android 11.

**Environment**
chatbot-builder version: [e.g., 1.0.0]
Kotlin version: [e.g., 1.7.0]
Android Gradle Plugin version: [7.4.2]
Gradle JDK version: [11.0.15]
OS: [macOS]

**Additional Info**
I have verified that the OpenAI API key is not empty, and the device is connected to the internet. I would appreciate any guidance or assistance in resolving this issue.

Thank you for your attention to this matter.

<img src="https://github.com/hexascribe/chatbot-builder/assets/68209547/790136d0-9c41-47c3-85ac-9dac7d11fd29" width=230  height=430/>

**Comment by osugikoji** [comment]
@osamasayed585 Hi, thank you for reporting. I wasn't able to reproduce it. It might be an issue with your OpenAI Key.

We have released a new feature that allows you to view the error logs. It's available in version 1.1.0. Could you please update to this new version and enable the log? This will help in identifying the issue you are facing.

To enable the error log, simply add the method .setLogError(true) in the builder class. For example:

```Kotlin
@Composable
fun ChatScreen() {
    val chatbot = ChatBot.Builder("MY_OPEN_AI_KEY")
        .setDarkMode(true)
        .addMessage(RoleEnum.ASSISTANT, "Hi, how can I help you today?")
        .addPreSeededMessage(RoleEnum.SYSTEM, "You are a helpful seller car assistant")
        .setInputFieldBorderWidth(1)
        .setLogError(true) // This will start showing any error in your logcat
        .build()
    chatBot.ComposeScreen()
}
```

**Comment by osugikoji** [comment]
Closing this because I was not able to reproduce it
